docs/src/content/docs/components/link-buttons.mdx

```diff
@@ -0,0 +1,152 @@
+---
+
+import { LinkButton } from '@astrojs/starlight/components';
+
+To display visually distinct call-to-action links, use the `<LinkButton>` component.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<LinkButton slot="preview" href="/getting-started/">
+	Read the docs
+</LinkButton>
+
+</Preview>
+
+## Import
+
+```tsx
+import { LinkButton } from '@astrojs/starlight/components';
+```
+
+## Usage
+
+Display a link button, useful for directing users to the most relevant or actionable content and often used on landing pages, using the `<LinkButton>` component.
+
+A `<LinkButton>` requires an [`href`](#href) attribute.
+Optionally, customize the appearance of the link button using the [`variant`](#variant) attribute, which can be set to `primary` (the default), `secondary`, or `minimal`.
+
+<Preview>
+
+```mdx
+# src/content/docs/example.mdx
+
+import { LinkButton } from '@astrojs/starlight/components';
+
+<LinkButton href="/getting-started/">Get started</LinkButton>
+<LinkButton href="/reference/configuration/" variant="secondary">
+	Configuration Reference
+</LinkButton>
+```
+
+<Fragment slot="markdoc">
+
+```markdoc
+# src/content/docs/example.mdoc
+
+{% linkbutton href="/getting-started/" %}Get started{% /linkbutton %}
+
+{% linkbutton href="/reference/configuration/" variant="secondary" %}
+Configuration Reference
+{% /linkbutton %}
+```
+
+</Fragment>
+
+<Fragment slot="preview">
+	<LinkButton href="/getting-started/">Get started</LinkButton>
+	<LinkButton href="/reference/configuration/" variant="secondary">
+		Configuration Reference
+	</LinkButton>
+</Fragment>
+
+</Preview>
+
+### Add icons to cards
+
+Include an icon in a link button using the [`icon`](#icon) attribute set to the name of [one of Starlight’s built-in icons](/reference/icons/#all-icons).
+
+The [`iconPlacement`](#iconplacement) attribute can be used to place the icon before the text by setting it to `start` (defaults to `end`).
+
+<Preview>
+
+```mdx {8-9}
+# src/content/docs/example.mdx
+
+import { LinkButton } from '@astrojs/starlight/components';
+
+<LinkButton
+	href="https://docs.astro.build"
+	variant="secondary"
+	icon="external"
+	iconPlacement="start"
+>
+	Related: Astro
+</LinkButton>
+```
+
+<Fragment slot="markdoc">
+
+```markdoc {6-7}
+# src/content/docs/example.mdoc
+
+{% linkbutton
+   href="https://docs.astro.build"
+	 variant="secondary"
+	 icon="external"
+	 iconPlacement="start" %}
+Related: Astro
+{% /linkbutton %}
+```
+
+</Fragment>
+
+<LinkButton
+	slot="preview"
+	href="https://docs.astro.build"
+	variant="secondary"
+	icon="external"
+	iconPlacement="start"
+>
+	Related: Astro
+</LinkButton>
+
+</Preview>
+
+## `<LinkButton>` Props
+
+**Implementation:** [`LinkButton.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/user-components/LinkButton.astro)
+
+The `<LinkButton>` component accepts the following props and also any other `<a>` attributes:
```

Comment:
I think it would be nice to link here like the `LinkCard` page does:

```suggestion
The `<LinkButton>` component accepts the following props and also any [other `<a>` attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a):
```